torch/nn/modules/transformer.py

```diff
@@ -235,6 +240,8 @@ def forward(self, src, src_mask=None):
 
 class TransformerDecoderLayer(Module):
     r"""TransformerDecoderLayer is made up of self-attn, multi-head-attn and feedforward network.
+        This standard decoder layer is based on the paper "Attention Is All You Need".
```

Comment:
Reference the note with the full citation for completion.

Reply:
added